runtime/ts/arc.ts

```diff
@@ -46,27 +46,27 @@ type PlanCallback = (recipe: Recipe) => void;
 type SerializeContext = {handles: string, resources: string, interfaces: string, dataResources: Map<string, string>};
 
 export class Arc {
-  private _context: Manifest;
-  private pecFactory: (id: string) => PECInnerPort;
-  private speculative: boolean;
+  private readonly _context: Manifest;
```

Comment:
Why do we still have some private member data with `_` prefix, are these just stragglers?

Reply:
If the private member is shadowed by a getter (i.e. readonly, except from the owning class) then we encode that with a `_` prefix. Not sure if that's the case here.